web/index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <!--
+    If you are serving your web app in a path other than the root, change the
+    href value below to reflect the base path you are serving from.
+
+    The path provided below has to start and end with a slash "/" in order for
+    it to work correctly.
+
+    For more details:
+    * https://developer.mozilla.org/en-US/docs/Web/HTML/Element/base
+
+    This is a placeholder for base href that will be replaced by the value of
+    the `--base-href` argument provided to `flutter build`.
+  -->
+  <base href="$FLUTTER_BASE_HREF">
+
+  <meta charset="UTF-8">
+  <meta content="IE=Edge" http-equiv="X-UA-Compatible">
+  <meta name="description" content="A new Flutter project.">
+
+  <!-- iOS meta tags & icons -->
+  <meta name="apple-mobile-web-app-capable" content="yes">
+  <meta name="apple-mobile-web-app-status-bar-style" content="black">
+  <meta name="apple-mobile-web-app-title" content="multiview_sample">
+  <link rel="apple-touch-icon" href="icons/Icon-192.png">
+
+  <!-- Favicon -->
+  <link rel="icon" type="image/png" href="favicon.png"/>
+
+  <title>multiview_sample</title>
+  <link rel="manifest" href="manifest.json">
+
+  <script>
+    // The value below is injected by flutter build, do not touch.
+    const serviceWorkerVersion = null;
+  </script>
+  <!-- This script adds the flutter initialization JS code -->
+  <script src="flutter.js" defer></script>
+</head>
+<body>
+  <div style="width: 100%; height: 100%;">
+    <div id="left" style="width: 50%; height: 100%; float: left;"></div>
+    <div id="right" style="width: 50%; height: 100%; float: left;"></div>
+  </div>
+  <script>
+    window.addEventListener('load', function(ev) {
+      // Download main.dart.js
+      _flutter.loader.loadEntrypoint({
+        serviceWorker: {
+          serviceWorkerVersion: serviceWorkerVersion,
+        },
+        onEntrypointLoaded: function(engineInitializer) {
+          engineInitializer.initializeEngine({
+            multiViewEnabled: true,
+          }).then(function(appRunner) {
+            appRunner.runApp().then(function(flutterApp) {
+              flutterApp.addView({hostElement: document.querySelector("#left")});
+              flutterApp.addView({hostElement: document.querySelector("#right")});
+            });
```

Comment:
For my own understanding: The screenshot in the PR description shows 3 views, this only adds two. Is the screenshot just outdated? Or does the web engine still utilize the implicit view as third view? If its the implicit view, what div is that one rendered into?

Reply:
The engine always creates an implicit view (which always has id `0`). It creates the element which hosts the flutter app dynamically once the app starts. I don't think we currently allow creating an app without creating the implicit view.

Reply:
Makes sense. We currently do the same on the other desktop platforms with plans to add a flag to disable the implicit view.

Reply:
We'll definitely disable the implicit view when `multiViewEnabled: true` :P

Reply:
The implicit view actually covers the two added views (try the `widgets_counter.dart` example). It just happens to work here because the app is mostly transparent.

Here, disabling the implicit view and making sure things actually run: https://github.com/flutter/engine/pull/48505

Reply:
And we need this too: https://github.com/goderbauer/mvp/pull/19